apps/golden-sample-app/src/app/services/journey-communication.service.spec.ts

```diff
@@ -19,8 +19,8 @@ describe('JourneyCommunicationService', () => {
 
   it('should add transfer to latests transactions', (done) => {
     service.latestTransaction$.subscribe((received) => {
-      expect(received?.transaction.amountCurrency.amount).toEqual(mockAmount);
-      expect(received?.merchant.name).toEqual(mocktoAccount);
+      expect(received?.transactionAmountCurrency.amount).toEqual(String(mockAmount));
```

Comment:
Why casting it?

Reply:
Because the typing is `latestTransactions$: Observable<TransactionItem | undefined>`